ha/core/controllers/pcs/pcs_controller.py

```diff
@@ -142,4 +142,11 @@ def nodes_status(self, nodeids: list) -> dict:
                     all_nodes_status[nodeid] = NODE_STATUSES.UNKNOWN.value
             else:
                 raise HAInvalidNode(f"Node {nodeid} is not a part of cluster")
+        for node in all_nodes_status.keys():
+            status = all_nodes_status[node]
+            if status.lower() == NODE_STATUSES.OFFLINE.value.lower() or \
+                status.lower() == NODE_STATUSES.UNKNOWN.value.lower():
```

Comment:
`status.lower() == NODE_STATUSES.UNKNOWN.value.lower():`
Not needed for unknown